packages/tools/kda-cli/.prettierignore

```diff
@@ -0,0 +1,4 @@
+lib/
+.kda-history.json
+tests/
+readme.md
```

Comment:
`lib` and `tests` is already be covered by the root .prettierignore.
Why do we not want to format `readme.md`